OpenNMT/train.py

```diff
@@ -258,11 +257,11 @@ def main():
         generator = nn.Sequential(
             nn.Linear(opt.rnn_size, dicts['tgt'].size()),
             nn.LogSoftmax())
-        if len(opt.gpus) > 1:
-            generator = nn.DataParallel(generator, device_ids=opt.gpus)
+#        if len(opt.gpus) > 1:
+#            generator = nn.DataParallel(generator, device_ids=opt.gpus)
         model = onmt.Models.NMTModel(encoder, decoder, generator)
         if len(opt.gpus) > 1:
-            model = nn.DataParallel(model, device_ids=opt.gpus)
+            model = nn.DataParallel(model, device_ids=opt.gpus, dim=1)
         if opt.gpus:
```

Comment:
Yeah, I'm hoping that functionality will be merged sooner rather than later. I'll give it a few days as I work on a better way to parallelize the generator. If it doesn't make it through by the time I finish, I guess I'll go back to dim=0 until it is merged.